### PR TITLE
Maps on Job View and Job Tasking now Click to Interact

### PR DESCRIPTION
**Pull request by lucanos**
First attempt to fix that pesky "*the map stole my scroll*" problem.

I create a transparent overlay and stick it in front of the map. If a user hovers over it, it becomes visible, along with instructions "Click to zoom or move map".
If the user clicks, the overlay disappears and the map can be scrolled/dragged/clicked.
When the user's mouse exists the map, the overlay resets.

### CODE REVIEW

File: Jobs/content/Tasking.js [hunk 1]
```diff
@@ -60,3 +60,22 @@ function initializeDateTimePicker(n, t) {
...
   $("#reportrange span").html(moment(n).format("MMMM D, YYYY H:mm") + " - " + moment(t).format("MMMM D, YYYY H:mm"))
 }
+
+var $lighthouse_mapblock;
+
+$(document).ready(function(){
+
+  $lighthouse_mapblock = $('<div id="lighthouse_mapblock"><div>Click to zoom or move map</div></div>');
+  $lighthouse_mapblock
+    .on('click',function(e){
```
Comment (sdunster) [inline Jobs/content/Tasking.js]: `.click(function(e) {`

File: Jobs/content/Tasking.js [hunk 2]
```diff
@@ -60,3 +60,22 @@ function initializeDateTimePicker(n, t) {
...
+      var $t = $(this);
+      $t.hide();
```
Comment (sdunster) [inline Jobs/content/Tasking.js]: You only use this once, why not `$(this).hide();`?

File: Jobs/content/View.js [hunk 3]
```diff
@@ -135,3 +135,22 @@ function taskFill(parent, child) {
...
   });
 }
+
+var $lighthouse_mapblock;
+
+$(document).ready(function(){
+
+  $lighthouse_mapblock = $('<div id="lighthouse_mapblock"><div>Click to zoom or move map</div></div>');
+  $lighthouse_mapblock
+    .on('click',function(e){
```
Comment (sdunster) [inline Jobs/content/View.js]: as in Tasking.js

File: Jobs/content/Tasking.js [hunk 4]
```diff
@@ -60,3 +60,22 @@ function initializeDateTimePicker(n, t) {
...
+      e.stopPropagation();
+    });
+  $('#map')
+    .append($lighthouse_mapblock)
+    .on('mouseleave',function(e){
```
Comment (sdunster) [inline Jobs/content/Tasking.js]: `.mouseleave()`

**Comment by sdunster** [comment]
Some nits. Over to @OSPFNeighbour to test and merge.

**Comment by OSPFNeighbour** [comment]
what about sectors page?

**Comment by lucanos** [comment]
Funky idea, Sam. I'll bear that one in mind next time.

**Pull request merged**